support-frontend/assets/helpers/subscriptionsForms/submit.ts

```diff
@@ -228,6 +228,7 @@ function buildRegularPaymentRequest(
 		csrUsername,
 		salesforceCaseId,
 		debugInfo: actionHistory,
+		similarProductsConsent: false, // TODO: We are only implementing this on the generic checkout for now
```

Comment:
Does this mean nothing through the old checkout will get this consent?

Reply:
As discussed we are going to change this to true to keep the current behaviour